fuzz/Cargo.toml

```diff
@@ -19,10 +21,25 @@ libfuzzer-sys = "0.4.0"
 target-lexicon = "0.12"
 wasmtime = { path = "../crates/wasmtime" }
 wasmtime-fuzzing = { path = "../crates/fuzzing" }
+component-test-util = { path = "../crates/misc/component-test-util", optional = true }
+
+[build-dependencies]
+anyhow = "1.0.19"
+proc-macro2 = "1.0"
+arbitrary = { version = "1.1.0", features = ["derive"] }
+rand = { version = "0.8.0" }
+quote = "1.0"
+wasmtime-fuzzing = { path = "../crates/fuzzing" }
```

Comment:
I'm a little worried about the impact that this will have on build times because it means that `cargo fuzz build` will build all of Wasmtime once for the host and then once again for the "target". This is a case where from Cargo's perspective when you execute `cargo build --target $the_host_target` it's considered a cross-compilation and packages shared between the two are built twice.

Improving this situation though is likely to not really be that easy. The "easy" thing to do is to split out the bits you need from `wasmtime-fuzzing` into their own separate crate. Splitting crates though is not always the easiest and ideally the pieces you're using to generate test cases here would ideally reuse the bits from `wasmtime-environ` as well to avoid duplicating things.

Overall though I do think that this is something that should be solved before landing due to how heavyweight this dependency is when fuzzing. This'll link in `v8`, for example, to the build script which is no small feat.

Reply:
This wasn't too painful to address.  See my latest commit.